### DISCUSSION TITLE
Just wanted to leave some kudos

### DISCUSSION THREAD

**Discussion by HannesHo** [andesse/hue-clip-api.node-red-flows#2] [General]
Thank you for your work! I was just about to do this on my own, you saved me a lot of time.
I was about to add a comment for color temp, but you already did. 
I was able to convert all my stuff from Huemagic. Great.

**Comment by andesse** [comment]
> Thank you for your work! I was just about to do this on my own, you saved me a lot of time. I was about to add a comment for color temp, but you already did. I was able to convert all my stuff from Huemagic. Great.

@HannesHo Awesome! That is exactly what I made it for :) Thanks!
